### PR TITLE
[MavenV2, MavenV3, XamariniOSV2] update used packages

### PR DESCRIPTION
**Pull request by DenisRumyantsev**
**Tasks names**: `MavenV2` , `MavenV3` , `XamariniOSV2`

**Description**: these tasks have been moved to this separate pull request from [another one](https://github.com/microsoft/azure-pipelines-tasks/pull/15807).

**Documentation changes required:** No

**Added unit tests:** No

[Attached related issue](https://github.com/microsoft/build-task-team/issues/2206)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

Changes have been tested by building tasks and uploading them to the Azure DevOps organization (using the `tfx` tool).
The test pipeline (that includes these tasks) has been created and run. The build has been passed successfully.

### CODE REVIEW

**Comment by tatyana-kostromskaya** [comment]
LGTM, tested these tasks manually on my own Azure DevOps organization.

**Review by tatyana-kostromskaya — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]

File: Tasks/MavenV2/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "azure-pipelines-tasks-codeanalysis-common": "^2.0.0-preview.0",
     "azure-pipelines-tasks-codecoverage-tools": "^2.0.0-preview.0",
     "azure-pipelines-tasks-java-common": "^2.0.0-preview.0",
-    "azure-pipelines-tasks-packaging-common": "^2.1.0",
+    "azure-pipelines-tasks-packaging-common-v3": "3.200.0",
```
Comment (KonstantinTyukalov) [inline Tasks/MavenV2/package.json:18]: Please pin the major version of the packages using `^`

File: Tasks/MavenV3/package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "azure-pipelines-tasks-codeanalysis-common": "2.0.0-preview.0",
     "azure-pipelines-tasks-codecoverage-tools": "2.0.0-preview.0",
     "azure-pipelines-tasks-java-common": "2.198.1",
-    "azure-pipelines-tasks-packaging-common": "^2.1.0",
+    "azure-pipelines-tasks-packaging-common-v3": "3.200.0",
```
Comment (KonstantinTyukalov) [inline Tasks/MavenV3/package.json:17]: Please pin the major version of the packages using `^`

File: Tasks/XamariniOSV2/package.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
     "@types/node": "^10.17.0",
     "@types/uuid": "^8.3.0",
     "azure-pipelines-task-lib": "^3.1.2",
-    "azure-pipelines-tasks-msbuildhelpers": "^2.198.1"
+    "azure-pipelines-tasks-msbuildhelpers-v3": "3.198.0"
```
Comment (KonstantinTyukalov) [inline Tasks/XamariniOSV2/package.json:24]: Please pin the major version of the packages using `^`

**Review by KonstantinTyukalov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**